pkg/fixtures/loader.go

```diff
@@ -6,48 +8,46 @@ import (
 	"github.com/applike/gosoline/pkg/mon"
 )
 
-type FixtureSet struct {
-	Enabled  bool
-	Writer   FixtureWriterFactory
-	Fixtures []interface{}
+func NewFixtureLoader(config cfg.Config, logger mon.Logger) FixtureLoader {
+	return &fixtureLoader{
+		config: config,
+		logger: logger,
+	}
 }
 
-type FixtureLoader struct {
-	fixtureSets []*FixtureSet
+type fixtureLoader struct {
+	config cfg.Config
+	logger mon.Logger
 }
 
-func NewFixtureLoader(fixtureSets []*FixtureSet) *FixtureLoader {
-	return &FixtureLoader{
-		fixtureSets: fixtureSets,
-	}
+type fixtureLoaderSettings struct {
+	Enabled bool `cfg:"enabled" default:"false"`
 }
 
-func (f *FixtureLoader) Load(config cfg.Config, logger mon.Logger) error {
-	logger = logger.WithChannel("fixture_loader")
+func (f *fixtureLoader) Load(fixtureSets []*FixtureSet) error {
+	settings := fixtureLoaderSettings{}
+	f.config.UnmarshalKey("fixtures", &settings)
 
-	if !config.IsSet("fixture_loader_enabled") {
-		logger.Info("fixture loader is not configured")
-		return nil
-	}
+	f.logger = f.logger.WithChannel("fixture_loader")
```

Comment:
This belongs in the NewFixtureLoader func, right?

Reply:
✅